Assets/Scripts/RagdollController.cs

```diff
@@ -53,8 +62,35 @@ public void EnableRagdoll(Vector3 knockbackForce)
         rigidbodyToPush.AddForce(knockbackForce, ForceMode.Impulse);
     }
 
-    public void ReparentCamera(Transform cameraTransform)
+    public void ReparentCamera(Camera camera, Transform input)
```

Comment:
This doesn't reparent the camera anymore, perhaps rename it?

Reply:
Will do!